packages/odyssey-storybook/.storybook/components/TokenTables.tsx

```diff
@@ -363,8 +367,7 @@ export const TokenTables = (): ReactNode =>
   tokenTables.map(({ name, values }) => (
     <DataTable
       columns={columns}
-      data={values}
-      fetchDataFn={() => values}
+      getData={() => values}
```

Comment:
DataTable is still in Labs, so we don't guarantee a stable API! Since this PR officially brings it out of Labs, _now_ we'll be making breaking changes.

Reply:
Yeah, makes sense. Disregard